recipes/magithub

```diff
@@ -0,0 +1,4 @@
+(magithub :fetcher github
+          :repo "vermiculus/magithub"
+          :files ("lisp/magithub.el"
```

Comment:
Please use wildcard or put it at top directory and use `:defaults`(Please see recipe format document(https://github.com/melpa/melpa#recipe-format)). This configuration let us updating recipe if you add other files in the future.